backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -132,15 +131,34 @@ def is_record_tree_selected(record, tree, is_all_selected: bool):
         tree["git_commit_hash"] == record["build__checkout__git_commit_hash"]
 
 
-def update_issues(record, task, is_failed_task):
-    issue_id = record["build__incidents__issue__id"]
-    if issue_id:
+# TODO unify with treeDetails
+def update_issues(
+    issue_id: Optional[str],
+    incident_test_id: Optional[str],
+    build_valid: Optional[bool],
+    issue_comment: Optional[str],
+    issue_report_url: Optional[str],
+    task,
+    is_failed_task: bool,
+    tab: str,
```

Comment:
nit: tab is a concept of frontend, so is a little weird use here. `from` appears to be a more suggestive name to this param

Reply:
`from` is a keyword, I'll use `issue_from`